applications/DG-Max/Algorithms/DivDGMaxHarmonic.cpp

```diff
@@ -198,14 +204,13 @@ void DivDGMaxHarmonic<DIM>::writeTec(std::string fileName) const {
 
 template <std::size_t DIM>
 double DivDGMaxHarmonic<DIM>::computeL2Error(
-    const typename DivDGMaxDiscretization<DIM>::InputFunction& exactSolution)
-    const {
+    const typename DivDGMaxDiscretization<DIM>::InputFunction& exactSolution) {
     return discretization_.computeL2Error(mesh_, 0, exactSolution);
 }
 
 template <std::size_t DIM>
 double DivDGMaxHarmonic<DIM>::computeL2Error(
-    const ExactHarmonicProblem<DIM>& problem) const {
+    const ExactHarmonicProblem<DIM>& problem) {
```

Comment:
the `const` is gone because of caching?

Reply:
It is because of moving the `Integral` classes up. They use `PhysicalElement/Face` for caching the basis functions and indeed they aren't const.